### ISSUE TITLE
Idea: Wikidata correspondence

### ISSUE THREAD

**Issue by mlinksva** [github/government-open-source-policies#26]
Some policies cataloged in this dataset have associated items in Wikidata, for example https://www.wikidata.org/wiki/Q43081078 is the item for the National Defense Authorization Act for Fiscal Year 2018. It could be beneficial to add an optional column for a corresponding wikidata item. Possible benefits:

- Wikidata might have other identifiers for policies, or simply other info about policies that is out of scope for this dataset, but may be useful to researchers, e.g., to analyze in conjunction with other datasets
- Wikidata might have some statements about policies which are also present in this dataset, and could be used to validate this dataset
- It's also possible that this dataset could be used to selectively create Wikidata items that should exist, but don't yet

This would probably take a fair amount of work to do well, and including a recurring job to check for moved/deleted/updated Wikidata items, though there's probably prior art to draw from other Wikidata<->domain-specific-database projects.

Briefly discussed with @Ainali ([Q23899609](https://www.wikidata.org/wiki/Q23899609) 😄) at the The EU Open Source Policy Summit 2023 one month ago.

**Comment by mlinksva** [comment]
Not necessary for <-> idea above, but seems like a good place to note item corresponding to this dataset https://www.wikidata.org/wiki/Q116972311

**Comment by Ainali** [comment]
> * It's also possible that this dataset could be used to selectively create Wikidata items that should exist, but don't yet

Yes, from a Wikidata notability perspective, I am pretty certain that every policy in this dataset could have a Wikidata item.

It might be worth noting that Wikidata has these two properties:

- [free/libre open source software development policy URL](https://www.wikidata.org/wiki/Property:P9904)
- [free/libre open source software usage policy URL](https://www.wikidata.org/wiki/Property:P9771)

who are meant to be used when one organization has their own policy for FOSS. If there are such policies in this dataset, besides having storing the URL at the organization item in Wikidata, that statement could get a reference using the [stated in](https://www.wikidata.org/wiki/Property:P248) property to link to the item for the policy.
